scripts/clang/format_code.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+if [ -z "${BUILD_PATH}" ];
+then
+  source "$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)/../set_vars.sh"
+fi
+
+cmake -B "${BUILD_PATH}" -DSC_CLANG_FORMAT_CODE=ON
+cmake --build "${BUILD_PATH}" --target clangformat
```

Comment:
I have some files formatted after I run this
![image](https://github.com/user-attachments/assets/7fd01b3f-a130-4672-a173-01ba84639e38)
don't you have it?

Reply:
reran formatting, possibly forgot it after changes last time